src/components/pages/dashboard/dashboard.js

```diff
@@ -81,7 +83,11 @@ export class Dashboard extends Component {
           .flatMap(_ => TelemetryService.getTelemetryByDeviceIdP1M(deviceIds))
       )
       .flatMap(transformTelemetryResponse(() => this.state.telemetry))
-      .map(telemetry => ({ telemetry, telemetryIsPending: false })); // Stream emits new state
+      .map(telemetry => ({ telemetry, telemetryIsPending: false })) // Stream emits new state
+      // Classify errors as retryable or not
+      .catch(error => Observable.throw(classifyError(error)))
```

Comment:
I don't think you need this line. If there is an error at this point, the error will already have been classified in the HttpClient. I think classifying it here might actually break the error logic in the panels (by changing the structure of the error message).